AuthenticatorBridgeKit/Tests/AuthenticatorBridgeItemDataTests.swift

```diff
@@ -16,19 +17,22 @@ final class AuthenticatorBridgeItemDataTests: AuthenticatorBridgeKitTestCase {
 
     override func setUp() {
         super.setUp()
+        cryptoService = MockSharedCryptographyService()
```

Comment:
In your existing tests, can you now check to see if the cryptography service decrypt and encrypt are called on the mock?

Reply:
Yeah, I definitely test that in the `AuthenticatorBridgeItemServiceTests` since the service has the responsibility. I didn't add that here since it's somewhat out of the scope of the `AuthenticatorBridgeItemData`. But since it's also running through the service to test the data object, I don't have an objection to adding that here as well.

Reply:
Added in the latest commit 👍